pkg/vm/engine/types.go

```diff
@@ -250,6 +250,9 @@ type Relation interface {
 	NewReader(context.Context, int, *plan.Expr, [][]byte) ([]Reader, error)
 
 	TableColumns(ctx context.Context) ([]*Attribute, error)
+
+	//max and min values
+	MaxAndMinValues(ctx context.Context) ([][2]any, []uint8, error)
```

Comment:
I think this is not a reasonable approach, min, max by query to get is the most appropriate.

Reply:
ok, thanks